app/controllers/login/send-again-post-controller.js

```diff
@@ -11,13 +11,17 @@ const CORRELATION_HEADER = require('../../utils/correlation_header').CORRELATION
 
 module.exports = (req, res) => {
   const correlationId = req.headers[CORRELATION_HEADER] || ''
-  userService.sendOTP(req.user, correlationId).then(
-    () => {
-      res.redirect(paths.user.otpLogIn)
-    },
-    (err) => {
-      errorView(req, res)
-      logger.error(err)
-    }
-  )
+  if (req.user.secondFactor === 'SMS') {
+    userService.sendOTP(req.user, correlationId).then(
+      () => {
+        res.redirect(paths.user.otpLogIn)
+      },
```

Comment:
would be clearer with an explicit `.catch()`. This is the pattern we've used elsewhere.